FunctionalTableData/Separator.swift

```diff
@@ -98,6 +119,9 @@ public extension UIView {
 		separator.constrainToBottomOfView(self)
 	}
 	
+	/// Removes any instance of a `Separator` view from the current view.
+	///
+	/// - Parameter withTag: the separator to remove.
```

Comment:
Just wondering since _ ; this parameter is not labeled for the consumer of this API - does it make sense to label it with the internal name in the docs?

Reply:
Yeah, they do (same example, official docs)

<img width="559" alt="screen shot 2018-05-17 at 9 51 23 am" src="https://user-images.githubusercontent.com/24159/40181768-e3f5739e-59b7-11e8-9440-4f8c1aa5dff2.png">